packages/flame_riverpod/example/README.md

```diff
@@ -0,0 +1,16 @@
+# example
```

Comment:
Update example readme with just a short comment about what it is showing, one or two sentences is enough.